todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/statistics/StatisticsViewModel.kt

```diff
@@ -75,17 +107,30 @@ class StatisticsViewModel(
      * Called when new data is ready.
      */
     private fun computeStats(tasks: List<Task>) {
-        numberOfCompletedTasks = tasks.count { it.isCompleted }
-        numberOfActiveTasks = tasks.size - numberOfCompletedTasks
+        var completed = 0
+        var active = 0
+
+        for (task in tasks) {
```

Comment:
consider a `foreach`

Reply:
tasks.forEach {  task -> is actually longer.